src/Views/Pages/Privacy.elm

```diff
@@ -12,43 +12,77 @@ import Views.Footer exposing (footerContent)
 privacyContent : Html Msg
 privacyContent =
     div [ class "section section--lighter" ]
-        [ h2 [] [ text "Privacy Policy and Cookies" ]
+        [ h2 [] [ text "Contemplating Action Privacy Policy and Cookies" ]
         , div []
             [ p []
-                [ text """
+                [ text """This policy relates to domain (“the Site”) which is owned by Neontribe whose registered address is lincoln street address (registered company XXX) and related content.
```

Comment:
This will be easier to read if you split on to more lines.... in the code. General rule of thumb is try to keep lines less than 80 columns.

No problem though.